btcjson/chainsvrcmds.go

```diff
@@ -363,6 +363,59 @@ func NewGetChainTipsCmd() *GetChainTipsCmd {
 	return &GetChainTipsCmd{}
 }
 
+// Int32OrNil defines a type for representing nullable numeric JSON-RPC parameters.
+// This is a way to explicitly marshall a null param as they are dropped by default.
+//
+// Useful when default value is unknown client-side and null param
+// needs to be included to define following params.
+type Int32OrNil struct {
+	Value *int32
+}
```

Comment:
Agreed it's a bit bloaty and it's not obvious why it was needed. The reason for `Int32OrNil` is to allow providing the JSON `null` value from command line in the following format:
```
btcctl getchaintxstats null 0000000000000000000039128c26efb79f720e54e4f741171c41b05debde0cdf
```

It allows user to provide `blockhash` while also providing `nblocks` as `null`. Possible options using `*int32`:
* Define a default `nblocks`
  * `nblocks` should be calculated server side with server's chain parameters
  * Reasonable default could be `4320` which is the server default in mainnet Bitcoin Core 
* Allow providing `null` for `*int32` in command parsing
  * `getchaintxstats` seems to be an exception so I avoided large changes to command parsing
* Forcing command line users to define `nblocks` to be able to define `blockhash`
* Asking user to provide e.g. `0` for `null` value

These are the options that I found. What do you suggest?

Reply:
Hmm yeah I see the challenge here. I think the larger issue is that we don't support `null` for `*int32` parameters, and given there are a bunch of RPC commands which have optional integer parameters, we should try to support this at some point. For now, let's stick with a `*int32`, and we can have a default `jsonrpcdefault:"4320"`. In the meantime, I'll put up an issue for this, since right now we basically just use defaults which are not `null`, and also don't allow providing `null` for `*int32` in command parsing. If we eventually do have these features, we can then do a bit of a `btcjson` cleanup and new command implementations will be a lot more straightforward. Sound good?